public/manifest.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "Condition Markers",
-  "version": "1.0.2",
-  "manifest_version": 3,
+  "version": "1.0.3",
+  "manifest_version": 4,
```

Comment:
I'm not aware that there is a manifest_version beyond `1`

Reply:
Oh is that supposed to be for standardizing different versions of the package? I usually don't touch it but kgbergman incremented it after they merged my last PR so I figured they wanted it incremented again.

Reply:
I think this would be used if owlbear would change the way the manifest files work, so they could check the manifest version and now how to parse it. Because there is only one version i think this isn't even checked. I could be wrong though: https://docs.owlbear.rodeo/extensions/reference/manifest

Reply:
That makes sense to me. I changed the version to 1 in a new commit.